test/autoLoadAdapter.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+/**
+ * Auto load adapter for tests
+ *
+ * Copyright (c) 2016 Rain Agency.
+ * Licensed under the MIT license.
+ */
+
+const Dynasty = require('dynasty');
+const https = require('https');
+
+class AutoLoadAdapter {
+  constructor() {
+    const dynasty = Dynasty({
```

Comment:
doesn't dynasty require the aws credentials? that's the reason i'm using a different ORM in https://github.com/mediarain/voxa/blob/master/samples/accountLinking/services/userStorage.js#L12